src/Android/Xamarin.Android/Samples/GraphicsOverlay/AddGraphicsWithSymbols/AddGraphicsWithSymbols.cs

```diff
@@ -0,0 +1,171 @@
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.AddGraphicsWithSymbols
+{
+    [Activity]
+    public class AddGraphicsWithSymbols : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Create the graphics overlay
+        private readonly GraphicsOverlay _overlay = new GraphicsOverlay();
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Add graphics with symbols";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create the map
+            Map myMap = new Map(BasemapType.Oceans, 56.075844, -2.681572, 13);
+
+            // Add the map to the map view
+            _myMapView.Map = myMap;
+
+            // Add the graphics overlay to the map view
+            _myMapView.GraphicsOverlays.Add(_overlay);
+
+            // Call functions to create the graphics
+            CreatePoints();
+            CreatePolygon();
+            CreatePolyline();
+            CreateText();
+        }
+
+        private void CreatePoints()
+        {
+            // Create a red circle simple marker symbol
+            SimpleMarkerSymbol redCircleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, Color.Red, 10);
+
+            // Create graphics and add them to graphics overlay
+            var graphic = new Graphic(new MapPoint(-2.72, 56.065, SpatialReferences.Wgs84), redCircleSymbol);
+            _overlay.Graphics.Add(graphic);
+
+            graphic = new Graphic(new MapPoint(-2.69, 56.065, SpatialReferences.Wgs84), redCircleSymbol);
+            _overlay.Graphics.Add(graphic);
+
+            graphic = new Graphic(new MapPoint(-2.66, 56.065, SpatialReferences.Wgs84), redCircleSymbol);
+            _overlay.Graphics.Add(graphic);
+
+            graphic = new Graphic(new MapPoint(-2.63, 56.065, SpatialReferences.Wgs84), redCircleSymbol);
+            _overlay.Graphics.Add(graphic);
+        }
+
+        private void CreatePolyline()
+        {
+            // Create a purple simple line symbol
+            SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dash, Color.Purple, 4);
+
+            // Create a new point collection for polyline
+            var points = new PointCollection(SpatialReferences.Wgs84)
```

Comment:
I would recommend using the ArcGIS Runtime 'PointCollection' type rather than 'var':
PointCollection points = new Esri.ArcGISRuntime.Geometry.PointCollection(SpatialReferences.Wgs84)